tests/mock_tables/config_db.json

```diff
@@ -31,7 +31,8 @@
         "mtu": "9100",
         "tpid": "0x8100",
         "pfc_asym": "off",
-        "speed": "40000"
+        "speed": "40000",
+        "dhcp_rate_limit": "300"
```

Comment:
Is there a test with a different rate limit?

Reply:
Hi @Yarden-Z , this rate limit of 300 packets/sec is set in the mock table because this is the default value that we have set using DB migrator. These values are only the starting point of the tests, we have written test cases that take these default configurations and remove and change the rate limits to different values. You can check out the test cases in dhcp_rate_test.py